### DISCUSSION TITLE
Signal missed frequently

### DISCUSSION THREAD

**Discussion by ghost** [NightscoutFoundation/xDrip#2216] [General]
Signal missed frequently. I gave all possible permits to the program.
![Screenshot_2022-07-10-12-08-29-80](https://user-images.githubusercontent.com/109020881/178139342-2d0605ba-db0c-412d-b3e8-8124097598e5.jpg)
![Screenshot_2022-07-10-12-22-18-03](https://user-images.githubusercontent.com/109020881/178139343-6f5aa0b4-2bfd-4cb9-bb72-8334200907e6.jpg)

**Comment by Navid200** [comment]
Why are critical information left out of your system status page screenshot?
Transmitter days is a very important parameter.
Can you take a screenshot that shows the entire page including transmitter days and battery voltages?

**Comment by ghost** [comment]
![Screenshot_2022-07-10-14-28-15-51](https://user-images.githubusercontent.com/109020881/178143139-dca174e7-d8ea-417c-a74e-b174734a9e13.jpg)
![Screenshot_2022-07-10-14-28-26-56](https://user-images.githubusercontent.com/109020881/178143142-9d1df157-a84c-4dbf-bc8e-4b29f2ad23d8.jpg)
I am sorry. It is called Dexcom One in Lithuania. But it's the same as G6. I started using only yesterday.

**Reply by Navid200** [reply]
Are all the settings (xDrip and Android) in agreement with the following?

https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

**Reply by ghost** [reply]
Thank you. All settings were good. I only made a tick here. Disable Settings −> Less Common Settings -> Advanced Calibration -> Automatic Calibration.  
I rebooted the phone and I will watch it works.

**Reply by Navid200** [reply]
All the other settings were good?!

What phone is this then?  Brand and model please

**Comment by ghost** [comment]
Did not help. Last update before 30 minutes.. Yes all settings good. The phone is not popular :) Realme 8 pro.  During the month, I should update to Android 12. Maybe it will work better. Thanks for trying.

**Reply by Navid200** [reply]
Have a look at this also:
https://navid200.github.io/xDrip/docs/Intermittent.html

**Comment by Ezis22** [comment]
Hi ,sorry my english.phon xiaomi mi note 10lite,android12.sametimes new sensor this one - two days.dexcom one ![Screenshot_2022-07-14-09-59-36-753_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933560-c8e17270-8d9a-4ff5-9efa-32d7d7783059.jpg)
![Screenshot_2022-07-14-09-55-13-284_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933572-a5e0610b-08be-4224-bd1d-717b1081978a.jpg)
![Screenshot_2022-07-14-09-31-08-260_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933577-6bb1477b-5803-4da9-80b7-2bcbbab7b9e4.jpg)
![Screenshot_2022-07-14-08-54-53-861_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933583-cc90bc81-1b7b-4686-b11c-4d9120e4bad1.jpg)
![Screenshot_2022-07-14-08-27-07-582_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933586-d0d87b8d-a9a6-49e7-9f7a-4c249eeb6741.jpg)
![Screenshot_2022-07-14-08-14-47-478_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933590-f4a11ca9-459a-4a43-80d1-b41c735eb699.jpg)
![Screenshot_2022-07-14-08-14-37-310_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/96720008/178933593-760e035f-e2ee-4e60-851f-295f6fa6f06b.jpg)

**Reply by Navid200** [reply]
These are the recommended settings to start from:  https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html
These are a few of the settings that can be tweaked depending on the phone: https://navid200.github.io/xDrip/docs/Intermittent.html

There are phones that make it very difficult for an app to run in the background.    There is no way to test every available device in the market.

**Reply by Ezis22** [reply]
Thanks.

**Comment by ghost** [comment]
Hi. I back after a long time. 
My observations over the past couple of years. This is not a Bluetooth problem.
Only after turning on "show on lock screen" did it start connecting every second time "10 min.".
How to force not to miss the session of connection?  
Every time before connecting, go to "G5/G6/DEX1 Debug Settings --> Adjust battery warning level" and press OK without changing anything. And connects without a problem. Why?
I can't do it all the time. That it could be done with Macro Droid ....

**Reply by Navid200** [reply]
This thread is about a very old Dexcom One device with the serial number starting with 5.
Do you still use such a device?
Or are you using one that starts with C now?

**Comment by ghost** [comment]
New sensors with "C" .But problems are the same.  My request. Please pay attention to it if you have time. It works if there is a problem with the connection. (Adjust battery warning level" and press OK). After this,  always connect. Checked with Xiaomi, Poco Realme.

**Reply by Navid200** [reply]
If you have found something and like to share it will be wonderful.
But, it has to be clear.

Reading your suggestion, it's not clear if you are suggesting that one can adjust the battery warning level only once and after that as long as the phone is on, there will never be a disconnect.
Is that what you are saying?
Because that is very easy to do and we can tell everyone to do it.  But, is that really what you are saying?
